<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserCLient</class>
 <widget class="QDialog" name="UserCLient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клиент</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>231</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbFIO">
      <property name="text">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leData"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbData">
      <property name="text">
       <string>Дата</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leFIO"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbPhone">
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lePhone"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbLogin">
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="leLogin"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbPassword">
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lePassword"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Поменять пароль?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>9</y>
     <width>109</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectAll">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>98</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать Все</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twClient">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>40</y>
     <width>751</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>246</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbClientID">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbID">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leID">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>lbClientID</zorder>
  <zorder>frame</zorder>
  <zorder>btnConnect</zorder>
  <zorder>btnSelectAll</zorder>
  <zorder>twClient</zorder>
  <zorder>btnExit</zorder>
  <zorder>lbID</zorder>
  <zorder>leID</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
